cli/build.rs

```diff
@@ -14,6 +14,12 @@ fn main() {
   let c = PathBuf::from(env::var_os("CARGO_MANIFEST_DIR").unwrap());
   let o = PathBuf::from(env::var_os("OUT_DIR").unwrap());
 
+  let root_names = vec![c.join("js/worker_globals.ts")];
+  let bundle = o.join("WORKER_SNAPSHOT.js");
+  let state = deno_typescript::compile_bundle(&bundle, root_names).unwrap();
+  assert!(bundle.exists());
+  deno_typescript::mksnapshot_bundle(&bundle, state).unwrap();
```

Comment:
How much bigger does it make the exe? And how much of a slow down to the build time?

Reply:
The run snapshot is 1.2Mb, but I'm pretty sure we'll be able to get away without needing to create third snapshot. #3731 has to be done first though